artiq/gateware/targets/sayma_amc.py

```diff
@@ -307,7 +309,7 @@ def __init__(self, jdcg_type, **kwargs):
         self.csr_devices.append("slave_fpga_cfg")
         self.config["SLAVE_FPGA_GATEWARE"] = 0x200000
 
-        rtio_channels = []
+        self.rtio_channels = rtio_channels = []
```

Comment:
Where is this used?

Reply:
Used by `add_std()` functions in `artiq.gateware.eem`